themes/ee/cp/js/build/components/relationship.js

```diff
@@ -258,10 +279,7 @@ function (_React$Component) {
         })),
         full: true,
         iframe: true,
-        dataType: 'json',
```

Comment:
I think this line might be needed? It's not in the source, but that could be mistake